### PR TITLE
daemon: remove bindata from git

### PR DESCRIPTION
**Pull request by aalemayhu**

### CODE REVIEW

File: daemon/Makefile [hunk 1]
```diff
@@ -8,6 +8,7 @@ GO_BINDATA := go-bindata -prefix ../ -mode 0640 -modtime 1450269211 \
 	-ignore Makefile -ignore bpf_features.h -ignore lxc_config.h \
 	-ignore netdev_config.h -ignore node_config.h -ignore '.+\.o$$' \
 	-ignore '.+\.orig$$' -ignore '.+~$$' -ignore '\.DS_Store'
+GO_BINDATA_SHA1SUM=4c07b9aaa78073a929aa67dc4bcb862de58d7522
```
Comment (tgraf) [inline daemon/Makefile]: Can you add a comment here that the value of this variable can be updated with `make -C daemon apply-bindata`?

File: daemon/Makefile [hunk 2]
```diff
@@ -9,6 +9,7 @@ GO_BINDATA := go-bindata -prefix ../ -mode 0640 -modtime 1450269211 \
 	-ignore netdev_config.h -ignore node_config.h -ignore '.+\.o$$' \
 	-ignore '.+\.orig$$' -ignore '.+~$$' -ignore '\.DS_Store'
 GO_BINDATA_SHA1SUM=4c07b9aaa78073a929aa67dc4bcb862de58d7522
+GO_VERSION_USED=go1.8.3
```
Comment (tgraf) [inline daemon/Makefile]: Is the expectation that we notice incompatible go versions by seeing this value changing in a patch?

**Review by tgraf — CHANGES_REQUESTED** [review]

File: daemon/Makefile (hunk 2 above)
Comment (aalemayhu) [reply daemon/Makefile]: Yes.

File: daemon/Makefile (hunk 1 above)
Comment (aalemayhu) [reply daemon/Makefile]: Done

**Review by tgraf — APPROVED** [review]
Awesome! I love this.

**Pull request merged**